doc/y_cable/design_doc.md

```diff
@@ -40,8 +40,9 @@ Challenge: to provide an interface for Y cable to interact with PMON docker and
 - It makes sense to keep the mapping file in the sonic_y_cable package so that it can be updated in the same pull request when a vendor adds a new cable implementation. However, we cannot install data files outside of the Python directory via a wheel. Considering this we propose to make this a Python file which simply contains a 2D dictionary which can be used to look up the module path. If the file is part of the sonic_y_cable package, it will be installed in both the PMon container and the host
 
 For example
-```
-	/sonic_platform_common/sonic_y_cable/y_cable_vendor_mapping.py
+```python
```

Comment:
This shouldn't be in Python formatting, as it's basically a directory listing. Generic code block is appropriate here.

Reply:
fixed